pyflakes/checker.py

```diff
@@ -1773,6 +1761,7 @@ def DICT(self, node):
         # Complain if there are duplicate keys with different values
         # If they have the same value it's not going to cause potentially
         # unexpected behaviour so we'll not complain.
+        counter = collections.Counter
```

Comment:
this should fix the call, this line looks suspicious as written

Reply:
@asottile rev 2d8f8de does as you requested.